assets/training/vision/environments/automl-dnn-vision-gpu/environment.yaml

```diff
@@ -0,0 +1,11 @@
+image:
+  name: azureml/curated/automl-dnn-vision-gpu
```

Comment:
please start a PR to MCR to declare the image

Reply:
As per the discussion with Vijetha, this image is already added to mcr here: https://github.com/microsoft/mcr/blob/main/teams/azureml/base/curated-unlisted.yml

Reply:
I think that one is different

Reply:
Ahh, I see it. The CE one is [unlisted]/azureml/curated/azureml-automl-dnn-vision-gpu. Thanks!